std_msgs/Header header

geometry_msgs/Pose bar_red
geometry_msgs/Pose bar_green
geometry_msgs/Pose bar_blue

float64[36] covariance_red
float64[36] covariance_green
float64[36] covariance_blue
